### PR TITLE
Check hook is valid before getting width

### PR DESCRIPTION
**Pull request by miiizen**
Resolves: #21020

### CODE REVIEW

**Review by RomanPudashkin — APPROVED** [review]

**Comment by DmitryArefiev** [comment]
Tested #21020 on Win10, Mac13.6 - **FIXED**

**Comment by Jojo-Schmitz** [comment]
Confirmed, same fix helps in my 3.7 too ;-)

File: src/engraving/rendering/dev/chordlayout.cpp [hunk 1]
```diff
@@ -1889,7 +1889,8 @@ void ChordLayout::layoutChords1(LayoutContext& ctx, Segment* segment, staff_idx_
                         upOffset = maxDownWidth + adjSpace;
                         if (downHooks) {
                             bool needsHookSpace = (ledgerOverlapBelow || ledgerOverlapAbove);
-                            double hookSpace = topDownNote->chord()->hook()->width();
+                            Hook* hook = topDownNote->chord()->hook();
+                            double hookSpace = hook ? hook->width() : 0.0;
```
Comment (cbjeukendrup) [inline src/engraving/rendering/dev/chordlayout.cpp:1893]: Would a similar change be potentially necessary in line 1929?

Comment (miiizen) [reply src/engraving/rendering/dev/chordlayout.cpp:1893]: Yeah definitely a good idea

Comment (Jojo-Schmitz) [reply src/engraving/rendering/dev/chordlayout.cpp:1893]: for master and 4.2.1?

**Pull request merged**